hedera-node/configuration/mainnet/bootstrap.properties

```diff
@@ -1,2 +1,15 @@
 # This file needs to exist even though there aren't any properties in it. Do NOT put properties in this file.
 # Downstream processes depend on its existence.
+# This is exact replica of file 121 on mainnet. It will be used by mono-service for all the property overrides.
```

Comment:
```suggestion
# This is an exact replica of file 121 on mainnet and must match application.properties exactly. It will be used by mono-service for all the property overrides. This file will be deprecated when we deprecated using mono-service config and fully move to hedera-config. 
```